js/app.js

```diff
@@ -0,0 +1,108 @@
+/* Pedimos al usuario una frase para encriptarla */
+var sentence = prompt('Escriba una frase: ');
+/* Verifica si escribio algo */
+if (sentence === ' ') {
+  alert('Escribe una frase');
+}
+/* Creamos una funcion, para Cifrar una frase */
+function cipher(sentence, number) {
+  /* Verifica si escribio solo letras */
+  if (sentence === parseInt(sentence)) {
+    alert('Escribe solo letras');
+  } else {
+    /* Almacenamos los newCodeAsciiOfLetterUp y newCodeAsciiOfLetterDown */
+    var arrayOfCodeAsciiLetter = [];
+    /* Hacemos un bucle para extraer los caracteres de la sentence */
+    for (i = 0; i < sentence.length; i++) {
+      /* Extraemos una letter de la sentence de acuerdo a su posición */
+      var letter = sentence.charAt(i);
+      /* Verificamos si la letra es mayuscula o miniscula */
+      if (letter === letter.toUpperCase()) {
+        /* Devuelve el Codigo ASCII de la letra mayuscula */
+        var codeAsciiOfLetterUp = letter.charCodeAt(0);
+        /* Aplicando la formula nos devuelve el nuevo código ASCII */
+        newCodeAsciiOfLetterUp = (codeAsciiOfLetterUp - 65 + n) % 26 + 65;
+        /* Aplicando el metodo push introducimos los valores de
+      newCodeAsciiOfLetterUp en arrayOfCodeAsciiLetter*/
+        arrayOfCodeAsciiLetter.push(newCodeAsciiOfLetterUp);
+      } else {
+        /* Devuelve el Codigo ASCII de la letra minuscula */
+        var codeAsciiOfLetterDown = letter.charCodeAt(0);
+        /* Aplicando la formula nos devuelve el nuevo código ASCII */
+        newCodeAsciiOfLetterDown = (codeAsciiOfLetterDown - 97 + n) % 26 + 97;
+        /* Aplicando el metodo push introducimos los valores de
+        newCodeAsciiOfLetterDown en arrayOfCodeAsciiLetter*/
+        arrayOfCodeAsciiLetter.push(newCodeAsciiOfLetterDown);
+      }
+    }
+    /* Declaramos newSentence vacio, para que tome los valores de la concatenación */
+    var newSentence = [];
+    /* Hacemos el bucle, para extraer los codigos ASCII que estan en el index de reversArray */
+    for (b = 0; b < arrayOfCodeAsciiLetter.length; b++) {
+      /* Devuelve la letra que se encuentra en el nuevo codigo ASCII */
+      var newLetter = String.fromCharCode(arrayOfCodeAsciiLetter[b]);
+      /* Concatena las letras */
+      newSentence.push(newLetter);
+    }
+    if (typeof(sentence) === 'string' && sentence !== ' ') {
+      /* Mostramos al usuario la frase incriptada mediante una alert */
+      return alert('La frase incriptada sería: ' + newSentence.join(''));
+    }
+  }
+}
+/* Llamamos a la fución */
+cipher(sentence, 33);
+/* Pedimos al usuario una frase para encriptarla */
+var sentenceDecipher = prompt('Escriba la frase que desea decifrar: ');
+/* Verifica si escribio algo */
+if (sentenceDecipher === ' ') {
+  alert('Escribe una frase');
+}
+/* Creamos una funcion, para Cifrar una frase */
+function decipher(sentenceDecipher, number) {
+  /* Verifica si escribio solo letras */
+  if (sentenceDecipher === parseInt(sentenceDecipher)) {
```

Comment:
Esto le dirá si el string contiene solo un número, pero no si contiene letras y números. Es mejor usar un regular expression:
https://stackoverflow.com/questions/12778083/regex-with-space-and-letters-only